coretex/cli/modules/node.py

```diff
@@ -458,7 +472,7 @@ def initializeNodeConfiguration() -> None:
             errorEcho("If you wish to reconfigure your node, use \"coretex node stop\" command first.")
             return
 
-        stop()
+        stop(config["nodeId"])
```

Comment:
`nodeId` which is saved in `config` is of `str` type (that is what `registerNode` functions returns) but `stop` expects it to be `int`. Change `registerNode` to return a `int` properly.